### ISSUE TITLE
Increase clickable area for Block controls

### ISSUE THREAD

**Issue by pento** [WordPress/gutenberg#4223]
The block move and block ellipsis buttons both have 20x20 clickable areas - this feels a little easy to miss, particularly with the hover visual feedback being quite subtle.

For comparison, the similar buttons in Dropbox Paper have a 24x24 icon, with a 32x32 clickable area.

I suspect this will become more necessary when #3044 is implemented, as having UI that disappears if you twitch too much is wildly frustrating.

Also mentioned here: https://wordpress.org/support/topic/tested-for-a-couple-hours-today-feedback/

**Comment by afercia** [comment]
Worth mentioning the new WCAG 2.1 (going to happen _soon_) will require the target size to be at least 44 by 22 pixels for the AA level, with a few exceptions. See https://www.w3.org/TR/WCAG21/#target-size

**Comment by pento** [comment]
I _knew_ there was an accessibility issue, but I was looking through the WCAG 2.0 spec for it. Thanks for the link, @afercia. 🙂

**Comment by afercia** [comment]
Yeah 2.1 is not an official recommendation yet, it will be soon and at that point we should start addressing this issue also in core where all things need to be... bigger :)

**Comment by karmatosed** [comment]
Can we see in a PR what 44 x 22 would be like? I'd agree we should go with that, depending on what it looks like.

**Comment by aduth** [comment]
Initial tinkering in developer tools to show general appearance of `44x22`:

![image](https://user-images.githubusercontent.com/1779930/34780531-65ed8704-f5f1-11e7-9e60-1ae255d42ec4.png)

([Full Screenshot](https://user-images.githubusercontent.com/1779930/34780572-7ff8201e-f5f1-11e7-9629-e1e2883c4582.png))

![image](https://user-images.githubusercontent.com/1779930/34780521-5fdd3990-f5f1-11e7-98ec-6485a7c54e28.png)

(In latter case, we could still allow clickable area at those dimensions but reduce focus effect styling)

**Comment by afercia** [comment]
Update: the target size requirement has been moved to level AAA in the [Candidate Recommendation of WCAG 2.1](https://www.w3.org/TR/WCAG21/). Some background here: http://www.incobs.de/articles/items/target-size-fate.html

**Comment by jasmussen** [comment]
I can fix this, I need to address this as part of https://github.com/WordPress/gutenberg/issues/5400 anyway.

**Comment by karmatosed** [comment]
@jasmussen as this is being done as part of another issue should we close this one and wrap them into one?

**Comment by jasmussen** [comment]
Yes, all issues mentioned in this one will be fixed when #5400 is fixed, or vice versa. I'll close one of them. But the PR to address both is almost ready to go.
